### PR TITLE
migration: Add case about abort job on target host

### PR DESCRIPTION
**Pull request by cliping**
VIRT-XXXX - VM live migration - abort job by domjobabort - on target host - precopy/postcopy

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

 (1/4) type_specific.io-github-autotest-libvirt.migration.abort_precopy_migration.abort_by_domjobabort_on_target.with_precopy.p2p: PASS (419.74 s)
 (2/4) type_specific.io-github-autotest-libvirt.migration.abort_precopy_migration.abort_by_domjobabort_on_target.with_precopy.non_p2p: PASS (428.82 s)
 (3/4) type_specific.io-github-autotest-libvirt.migration.abort_precopy_migration.abort_by_domjobabort_on_target.with_postcopy.p2p: PASS (388.48 s)
 (4/4) type_specific.io-github-autotest-libvirt.migration.abort_precopy_migration.abort_by_domjobabort_on_target.with_postcopy.non_p2p: PASS (389.38 s)

**Comment by cliping** [comment]
modular_redesign

File: provider/migration/migration_base.py [hunk 1]
```diff
@@ -672,3 +672,25 @@ def get_NM_service(params=None, remote_host=False):
...
+    dest_uri = params.get("virsh_migrate_desturi")
+    vm_name = params.get("main_vm")
+    domjobabort_err_msg = params.get("domjobabort_err_msg")
+    postcopy_options = params.get("postcopy_options")
+
+    if postcopy_options:
+        ret = virsh.domjobabort(vm_name, option="--postcopy", debug=True, uri=dest_uri)
+    else:
+        ret = virsh.domjobabort(vm_name, debug=True, uri=dest_uri)
+    if ret.exit_status:
```
Comment (dzhengfy) [inline provider/migration/migration_base.py:692]: Is it ok to use `libvirt.check_result()`?

**Review by dzhengfy — COMMENTED** [review]
Others LGTM

File: provider/migration/migration_base.py (hunk 1 above)
Comment (cliping) [reply provider/migration/migration_base.py]: Updated, thanks.

**Review by dzhengfy — APPROVED** [review]
LGTM

**Review by Yingshun — APPROVED** [review]

**Pull request merged**